window/Cargo.toml

```diff
@@ -46,6 +46,7 @@ wezterm-bidi = { path = "../bidi" }
 wezterm-color-types = { path = "../color-types" }
 wezterm-font = { path = "../wezterm-font" }
 wezterm-input-types = { path = "../wezterm-input-types" }
+wezterm-term = { path = "../term", features=["use_serde"] }
```

Comment:
as suggested elsewhere, I think the mapping can be done in the wezterm-gui layer and that means that you can undo these changes to the window crate; in theory the window crate can be used to create non-terminal programs, so it is desirable to avoid pulling in a terminal emulator core from wezterm-term for that

Reply:
Yes, fixed in https://github.com/wez/wezterm/pull/6292/commits/b67cf84b159515d2789c946019763bb3dca61ed6 and https://github.com/wez/wezterm/pull/6292/commits/683d1348a35a558dc781faa135a4de71f8cea49c!